src/main/java/com/iota/iri/BundleValidator.java

```diff
@@ -8,8 +8,40 @@
 
 import java.util.*;
 
+/**
+ * Validates bundles.
+ * <p>
+ * Bundles are a list of transactions that represent an atomic transfer, meaning that either all
+ * transactions inside the bundle will be accepted by the network, or none.
+ *</p>
+ */
 public class BundleValidator {
 
+    /**
+     * Fetches a bundle of transactions identified by the {@code tailHash} and validates them.
+     * <p>
+     * The fetched transactions have the same bundle hash as the transaction identified by {@code tailHash}
+     * The validation ascertains that:
+     * <ol>
+     *     <li>{@code tailHash} has an index of 0</li>
+     *     <li>{@code tailHash} was not already invalidated by a previous call to this method</li>
+     *     <li>That the transactions reference order is consistent with the indexes</li>
+     *     <li>The last index of each transaction in the bundle matches the last index of the tail transaction</li>
+     *     <li>While summing up the bundle value we never exceed the total supply of iotas</li>
```

Comment:
absolute value should be used in the comment?

Reply:
:+1: